promptsource/templates/social_i_qa/templates.yaml

```diff
@@ -37,10 +61,11 @@ templates:
       _do_eval: false
       _do_train: true
       choices_in_prompt: false
-      metrics: []
+      metrics:
+      - Accuracy
       original_task: true
```

Comment:
Let's mark this as not original task for now? It recasts a 5-way classification to a binary classification which makes its accuracy incomparable to others.